### PR TITLE
[simd] Add spec tests for 64x2 lane ops

### PR DESCRIPTION
**Pull request by Honry**

### CODE REVIEW

**Comment by Honry** [comment]
@AndrewScheidecker, PTAL, thanks!

File: Test/spec/simd/simd_lane.wast [hunk 1]
```diff
@@ -320,17 +426,28 @@
...
+(assert_invalid (module (func (result v128) (f64x2.replace_lane 2 (v128.const f64x2 0 0) (f64.const 1.0)))) "invalid lane index")
 
 ;; Lane index is determined by the instruction's interpretation only.
 
 (assert_invalid (module (func (result i32) (i16x8.extract_lane_s 8 (v128.const i8x16 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0)))) "invalid lane index")
 (assert_invalid (module (func (result i32) (i16x8.extract_lane_u 8 (v128.const i8x16 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0)))) "invalid lane index")
 (assert_invalid (module (func (result i32) (i32x4.extract_lane 4 (v128.const i8x16 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0)))) "invalid lane index")
 (assert_invalid (module (func (result i32) (f32x4.extract_lane 4 (v128.const i8x16 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0)))) "invalid lane index")
-(assert_invalid (module (func (result v128) (f16x8.replace_lane 4 (v128.const i8x16 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0) (i32.const 1)))) "invalid lane index")
+(assert_invalid (module (func (result v128) (i16x8.replace_lane 4 (v128.const i8x16 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0) (i32.const 1)))) "invalid lane index")
```
Comment (AndrewScheidecker) [inline Test/spec/simd/simd_lane.wast]: This is failing with:
```
/home/travis/build/WAVM/WAVM/Test/spec/simd/simd_lane.wast:444:2: module was well formed and valid
(assert_invalid (module (func (result v128) (i16x8.replace_lane 4 (v128.const i8x16 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0) (i32.const 1)))) "invalid lane index")
 ^
```

I assume it should be `i16x8.replace_lane 8`?

Comment (Honry) [reply Test/spec/simd/simd_lane.wast]: Ops, I forgot to force push my fix in local before.  🤕

Comment (Honry) [reply Test/spec/simd/simd_lane.wast]: Done now.

**Pull request merged**